src/go/MONIT/annotationManager.go

```diff
@@ -850,6 +865,7 @@ func parseTimes(ifCreatingAnnotation bool) []int64 {
 func main() {
 
 	flag.StringVar(&annotation, "annotation", "", "Annotation text")
+	flag.StringVar(&grafanaLimit, "grafana-limit", "100", "Limit for fetching number of annotations at a time.")
```

Comment:
the limit is an integer type, once you need to pass it to a string you'll use %d in fmt.Sprinf

Reply:
fixed.